command/server/server.go

```diff
@@ -183,6 +184,17 @@ func setFlags(cmd *cobra.Command) {
 		"minimum block time in seconds (at least 1s)",
 	)
 
+	cmd.Flags().Uint64Var(
+		&params.rawConfig.IBFTBaseTimeout,
+		ibftBaseTimeoutFlag,
+		// Calculate from block time if it is not given
+		0,
```

Comment:
Please note that Cobra will print `(default: 0)` in the `--help` command for the flag

Reply:
What do you mean?

Reply:
This is not a constant default value

Reply:
If you run `server --help`, you'll see double default values printed out

Reply:
Fixed